src/Uno.UI/UI/Xaml/Shapes/Path.wasm.cs

```diff
@@ -30,7 +32,74 @@ protected override SvgElement GetMainSvgElement()
 						("d", gd.Data),
 						("fill-rule", gd.FillRule == FillRule.EvenOdd ? "evenodd" : "nonzero"));
 					break;
+				case PathGeometry pg:
+					_path.SetAttribute(("d", ToStreamGeometry(pg)));
+					break;
+			}
+		}
+		/// <summary>
+		/// Transform the figures collection into a SVG Path according to :
+		/// https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/d
+		/// </summary>
+		/// <param name="geometry"></param>
+		/// <returns></returns>
+		private string ToStreamGeometry(PathGeometry geometry)
+		{
+			StringBuilder sb = new StringBuilder();
+
+			foreach (PathFigure figure in geometry.Figures)
+			{
+				sb.Append("M " + figure.StartPoint.X + "," + figure.StartPoint.Y + " ");
```

Comment:
This will format using the current culture, `InvariantCulture` must be used.

Reply:
Are you going to push a new update >

Reply:
Yep, CI in progress :)